multinet/__init__.py

```diff
@@ -34,6 +34,9 @@ def create_app(config: Optional[MutableMapping] = None) -> Flask:
     """Create a Multinet app instance."""
     app = Flask(__name__)
 
+    if config is not None:
+        app.config.update(config)
```

Comment:
What does this do? Why wasn't it needed before now?

Reply:
Before, we had the `config` parameter in this function, but we were never using it. We need to use it to allow setting the `TESTING` flag, for test servers. https://github.com/multinet-app/multinet-server/blob/acc8864e8f7f114b15b5f380ce29540955957088/test/conftest.py#L13